packages/web-runtime/src/composables/upload/useUpload.ts

```diff
@@ -79,7 +79,18 @@ export function useUpload(options: UploadOptions): UploadResult {
       onBeforeRequest: (req) => {
         req.setHeader('Authorization', unref(headers).Authorization)
       },
-      headers: () => unref(headers),
+      headers: (file) => {
+        const reqHeaders = {
+          'x-oc-mtime': file.data.lastModified / 1000,
+          ...unref(headers)
+        } as uppyHeaders
+
+        if (unref(isTusSupported)) {
+          // Tus includes auth headers before each request
```

Comment:
wouldn't the TUS handling just overwrite the auth headers then? In that case I'd prefer not manipulating them here to keep the code clean(er).

Reply:
> wouldn't the TUS handling just overwrite the auth headers then?

Unfortunately not. The headers here overwrite the modifications in `beforeEachRequest()`.

Reply:
ah, ok :-)